e2e/ui/run.sh

```diff
@@ -30,6 +30,9 @@ EOF
 
 IMAGE="mcr.microsoft.com/playwright:v1.21.0-focal"
 
+realpath=$(which realpath || which grealpath) # macOS compat
```

Comment:
Does this come by default in macOS? I don't remember what I did, but for me it seems like it was something I installed externally 😅

Reply:
Oh shoot, this is wrong anyways... `realpath` exists in macOS it just has weird arguments. Lemme rework this.

Reply:
Bah no that's wrong. _I_ have `realpath` on my mac because I've installed it that way. So no, `grealpath` isn't usually there but at least then we'd get an error and can tell folks to install it.